TESTING-INSTRUCTIONS.md

```diff
@@ -130,6 +130,12 @@ In case the report shows "no data", please reimport historical data by following
 -   Click on the 3 dots of the card, click `Hide this`, it should make the card disappear, it should also not show on refresh.
     This can't be shown again unless the `woocommerce_show_marketplace_suggestions` option is deleted (through PHPMyAdmin or using `wp option delete woocommerce_show_marketplace_suggestions`).
 
+### Correct the left position of transient notices when the new nav is used #6914
```

Comment:
Since this PR is a small adjustment to UI, I'm leaning towards not including any test instructions at all

Reply:
Fair enough, I included test instructions because CSS changes tend to cause regressions, especially when tested in other browsers. But I'll remove them.